recipes/visp/bld.bat

```diff
@@ -0,0 +1,34 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+cd build
+
+set "CC=clang-cl.exe"
+set "CXX=clang-cl.exe"
+set "CL=/MP"
+
+::Configure
+cmake ^
+    %SRC_DIR% ^
+    -G Ninja ^
+    -DOGRE_DIR="%LIBRARY_PREFIX%\cmake" ^
+    -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DVISP_LIB_INSTALL_PATH:PATH="lib" ^
+    -DVISP_BIN_INSTALL_PATH:PATH="bin" ^
+    -DVISP_CONFIG_INSTALL_PATH:PATH="cmake" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DBUILD_TESTS=ON
```

Comment:
Nit: Grouping Windows specific-argument together (I do not know if one can put a comment to separate both groups of arguments without breaking the command line).
```suggestion
    -G Ninja ^
    -DBUILD_TESTS=ON ^
    -DOGRE_DIR="%LIBRARY_PREFIX%\cmake" ^
    -DCMAKE_BUILD_TYPE:STRING=Release ^
    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
    -DVISP_LIB_INSTALL_PATH:PATH="lib" ^
    -DVISP_BIN_INSTALL_PATH:PATH="bin" ^
    -DVISP_CONFIG_INSTALL_PATH:PATH="cmake" ^
    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX%
```